packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -488,12 +492,823 @@ function Checkbox(props) {
+hook from `@react-stately/table` is responsible for initializing and tracking the widths of every column in your table, returning functions that you can use to
+update the column widths during a column resize operation. Note that this state is supplementary to the state returned by <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableState} />.
+
+The second column resizing hook is <TypeLink links={docs.links} type={docs.exports.useTableColumnResize} />. This hook handles the interactions for the a table column's resizer
+element, allowing the user to drag the resizer or use the keyboard arrows to expand the column's width. Be sure to pass the state returned by <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} />
+to this hook so the tracked widths can be updated appropriately. We'll walk through all the changes to the previous table implementation step by step below.
+
+### Table
+
+As mentioned previously, we first need to call <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} /> to initialize and update the widths for our table's columns.
+By providing the hook with `getDefaultMinWidth` and `getDefaultWidth`, we can give our table selection checkbox column a pre-determined width while allowing the other columns divide up the remaining
+width equally initially. We'll pass the state returned by <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} /> along with any user defined `onResize` handlers
+to our `ResizableTableColumnHeaders` so it can be used by <TypeLink links={docs.links} type={docs.exports.useTableColumnResize} />. The `widths` from this state are provided to each table element so
+the table cells in the body match the columns at all times.
+
+The various style changes below are to make the table itself scrollable and to support table body/column widths greater than the 400px applied to the table itself.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+/*- begin highlight -*/
+import {useTableColumnResizeState} from '@react-stately/table';
+/*- end highlight -*/
+
+function ResizableColumnsTable(props) {
+  let {
+    selectionMode,
+    selectionBehavior,
+    /*- begin highlight -*/
+    onResizeStart,
+    onResize,
+    onResizeEnd
+    /*- end highlight -*/
+  } = props;
+  let state = useTableState({
+    ...props,
+    showSelectionCheckboxes: selectionMode === 'multiple' && selectionBehavior !== 'replace'
+  });
+
+  let ref = useRef();
+  let {collection} = state;
+  let {gridProps} = useTable(
+    {
+      ...props,
+      /*- begin highlight -*/
+      // The table itself is scrollable rather than just the body
+      scrollRef: ref
```

Comment:
Two possible strategies that I considered here. One is the current which is to make the table scrollable + sticky column headers (exactly how the aria example we have in the storybook is). Unfortunately this makes scrollIntoView iffy since it doesn't account for the sticky column headers taking room since it is floating on top of the scroll region.

Second strategy is to follow how we sync header scroll position in TableView. This felt a bit more complex for an aria example, but happy to implement if people think its ok